sdk/communication/communication-common/test/identifierModelSerializer.spec.ts

```diff
@@ -34,7 +34,7 @@ const assertThrowsMissingProperty = <
 ): void => {
   assert.throws(() => {
     deserializeCommunicationIdentifier(serializedIdentifier);
-  }, `Property ${missingPropertyName} is required for identifier of type ${identifierType}.`);
+  }, `Property ${String(missingPropertyName)} is required for identifier of type ${identifierType}.`);
```

Comment:
Why was this necessary all of a sudden?

Reply:
It was done because of the warning "Implicit conversion of a 'symbol' to a 'string' will fail at runtime. Consider wrapping this expression in 'String(...)'"

Reply:
@petrsvihlik It was done because of the warning "Implicit conversion of a 'symbol' to a 'string' will fail at runtime. Consider wrapping this expression in 'String(...)'"

Reply:
This seems to be something that has been introduced with TS 4.7. Folks from Google reported it here: https://github.com/microsoft/TypeScript/issues/48848

There might be [alternative ways](https://github.com/microsoft/TypeScript/issues/48848#issuecomment-1110089918) of fixing this like `Q extends string & keyof Required<SerializedCommunicationIdentifier>[P]`

I'd suggest one of the following:
- read the related GH issue, understand the problem, fix it accordingly
- undo the change and create a follow-up ADO item with all the details

Reply:
@petrsvihlik I rolled back and createn an ADO

Reply:
@petrsvihlik  [User Story 3024912](https://skype.visualstudio.com/SPOOL/_workitems/edit/3024912): [SDK][JS][Common] Fix "strictNullChecks" error warning